### PR TITLE
Feature entity board

### PR DESCRIPTION
**Pull request by tinnathuridur**
We're trying to create a new branch where we can track all the entity classes

### CODE REVIEW

**Comment by tinnathuridur** [comment]
removed problem files

File: .gradle/3.1/taskArtifacts/cache.properties [hunk 1]
```diff
@@ -1 +1 @@
-#Sat Oct 22 15:36:42 GMT 2016
+#Sat Oct 29 16:39:20 GMT 2016
```
Comment (sibbaros) [inline .gradle/3.1/taskArtifacts/cache.properties]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 2]
```diff
@@ -44,9 +44,9 @@ public static void fillBoard()
 	
 	/**
 	 *return the character for the selected place in the board
-	 *@return X or O if a player has selected the field, else empty string
 	 *@param x		the x coordinate of the board
 	 *@param y		the y coordinate of the board
+	 *@return X or O if a player has selected the field, else empty string
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:49]: merge conflict

**Review by sibbaros — APPROVED** [review]

**Review by sibbaros — APPROVED** [review]

File: src/main/java/entityClasses/Board.java [hunk 3]
```diff
@@ -77,8 +77,8 @@ public static void setPositionChar(int x, int y, char playerChar)
 	
 	/**
 	 *checks if the board has a win for our current Player
-	 *@return true if the current Player made a win move
 	 *@param playerChar		the character of the current player
+	 *@return true if the current Player made a win move
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:81]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 4]
```diff
@@ -132,8 +132,8 @@ public static boolean checkFullBoard()
 	
 	/**
 	 *get the right character for current player
-	 *@return X or O
 	 *@param playerChar		the character of the current player
+	 *@return X or O
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:136]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 5]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
 		return playerO;
 		
 	}
-	
-	
+
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:148]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 6]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+	/**
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:149]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 7]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+	 *checks if the space for the character is empty or not, returns bool
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:150]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 8]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+	 *@param x		the x coordinate of the board
+	 *@param y		the y coordinate of the board
+	 *@return a bool false if position on board is empty, true if not empty position 
+	 *@see boolean
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:154]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 9]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
 		return playerO;
 		
 	}
-	
-	
+
+	/**
+	 *checks if the space for the character is empty or not, returns bool
+	 *@param x		the x coordinate of the board
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:151]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 10]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+	 *@param y		the y coordinate of the board
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:152]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 11]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+	 *@return a bool false if position on board is empty, true if not empty position
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:153]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 12]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
-	
-	
+
+	/**
+	 *checks if the space for the character is empty or not, returns bool
+	 *@param x		the x coordinate of the board
+	 *@param y		the y coordinate of the board
+	 *@return a bool false if position on board is empty, true if not empty position 
+	 *@see boolean
+	 */
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:155]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 13]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+	public static boolean isOccupied(int x, int y)
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:156]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 14]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+	{
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:157]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 15]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+		if(boardArray[x][y] != ' ')
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:158]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 16]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+		{
+			//place on board is not empty, it is occupied
+			return true;
+		}
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:162]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 17]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+	 *checks if the space for the character is empty or not, returns bool
+	 *@param x		the x coordinate of the board
+	 *@param y		the y coordinate of the board
+	 *@return a bool false if position on board is empty, true if not empty position 
+	 *@see boolean
+	 */
+	public static boolean isOccupied(int x, int y)
+	{
+		if(boardArray[x][y] != ' ')
+		{
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:159]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 18]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+			//place on board is not empty, it is occupied
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:160]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 19]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+			return true;
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:161]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 20]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+		}
+		//place on board is empty, it is not occupied
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:163]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 21]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+		return false;
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:164]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 22]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+	}
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:165]: merge conflict

File: src/main/java/entityClasses/Board.java [hunk 23]
```diff
@@ -145,7 +145,24 @@ public static char getPlayer(char playerChar)
...
+
```
Comment (sibbaros) [inline src/main/java/entityClasses/Board.java:166]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 24]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
 		assertEquals(true, b.checkFullBoard());
 	}
 
+	@Test
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:78]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 25]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+	public void testOccupied()
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:79]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 26]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+	{
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:80]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 27]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+		Board b = new Board();
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:81]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 28]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+		//testing empty board to see if empty place on certain position
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:82]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 29]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+		//for testing comment
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:83]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 30]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+		assertEquals(false, b.isOccupied(0, 0));
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:84]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 31]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:85]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 32]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+		b.setPositionChar(0,0, 'O');
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:86]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 33]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+		//if position is occupied a true should return
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:87]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 34]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+		assertEquals(true, b.isOccupied(0,0));
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:88]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 35]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+	}
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:89]: merge conflict

File: src/test/java/entityClasses/BoardTest.java [hunk 36]
```diff
@@ -75,5 +75,18 @@ public void testFullBoard()
...
+
```
Comment (sibbaros) [inline src/test/java/entityClasses/BoardTest.java:90]: merge conflict

**Review by sibbaros — APPROVED** [review]

**Pull request closed without merging**